src/main/java/com/expediagroup/hiveberg/IcebergInputFormat.java

```diff
@@ -163,6 +167,7 @@ public boolean next(Void key, IcebergWritable value) {
           value.setRecord(currentRecord);
         } else {
           value.setRecord(recordWithVirtualColumn(currentRecord, currentSnapshotId, table.schema(), virtualSnapshotIdColumnName));
+          LOG.info("Set virtual column id to: " + currentSnapshotId);
```

Comment:
This is probably better at `debug` level, in normal operation we wouldn't want to see these each time a query is run.

Reply:
Ah yeah, that's a good point, i'll change this round